packages/plugin-calendar/src/index.js

```diff
@@ -0,0 +1,58 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+import '@ciscospark/plugin-wdm';
+import '@ciscospark/plugin-encryption';
+import '@ciscospark/plugin-conversation';
+
+import {registerPlugin} from '@ciscospark/spark-core';
+import Calendar from './calendar';
+import config from './config';
+import {has, get} from 'lodash';
+
+registerPlugin(`calendar`, Calendar, {
+  config,
+  payloadTransformer: {
+    predicates: [],
+    transforms: [
+      {
+        name: `decryptMeeting`,
+        direction: `inbound`,
+
+        fn(ctx, object) {
+          return ctx.spark.calendar.decryptMeeting(object)
```

Comment:
Instead of calling this function you need to make it so that the transforms below are called: https://github.com/ciscospark/spark-js-sdk/blob/master/packages/plugin-team/src/index.js#L20-L39